packages/fast-components-react-msft/src/action-trigger/action-trigger.schema.json

```diff
@@ -0,0 +1,37 @@
+{
+    "$schema": "http://json-schema.org/schema#",
+    "title": "Action trigger",
+    "description": "An action trigger component's schema definition.",
+    "type": "object",
+    "id": "action-trigger",
+    "properties": {
+        "disabled": {
```

Comment:
Is there no way to alter the glyph in the side pane?

Reply:
Not at the moment as we don't yet have an implementation for render props here.  Tracked by https://github.com/Microsoft/fast-dna/issues/1143